### ISSUE TITLE
 Assembly with flye failed

### ISSUE THREAD

**Issue by ps120195** [alekseyzimin/masurca#224]
Hi,
I am trying to run Masurca for Hybrid Assembly of ONT with illumina short reads, I am getting this below error after running for some time 

Fri Feb 26 11:59:35 UTC 2021] Output prefix mr.41.17.12.0.02
[Fri Feb 26 11:59:35 UTC 2021] Pre-correction and initial filtering of the long reads
[Fri Feb 26 12:26:55 UTC 2021] Failed to pre-correct .fastq.gz file, please check your data!
[Fri Feb 26 12:26:55 UTC 2021] Assembly with flye failed, see files under flye/

I checked the path of my file in config,txt, it is correct.
I already have done long read assembly with the same nanopore data but why I am getting this pre-correct step error in Masurca?
Please help

**Comment by bdentinger** [comment]
check the .err file. i had this issue and it was because i needed to install numactl.

**Comment by aakashsur** [comment]
Yes I ran into the same issue, this should probably be a check in `configure`

**Comment by asgiraldoc** [comment]
> check the .err file. i had this issue and it was because i needed to install numactl.

Yes, that worked for me :D thank you

**Comment by BytePers** [comment]
I had a similar problem. Changing the parameter LHE_COVERAGE=25 to LHE_COVERAGE=30 solved it.
